plugins/modules/ndo_qos_dscp_cos_translation_policy.py

```diff
@@ -0,0 +1,555 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: ndo_qos_dscp_cos_translation_policy
+short_description: Manage QoS DSCP CoS Translation Policies on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Quality of Service (QoS) Differentiated Services Code Point (DSCP) Class of Service (CoS) Translation Policies.
```

Comment:
Looks like users can only configure one QoS DSCP CoS Translation Policy per template. There is a similar restriction on MCP Global Policies. I added a few notes to indicate this and that updating a policy with a different name would only update the existing policy instead of adding a new one. eg [L20](https://github.com/CiscoDevNet/ansible-mso/blob/master/plugins/modules/ndo_mcp_global_policy.py#L20) and [L111](https://github.com/CiscoDevNet/ansible-mso/blob/master/plugins/modules/ndo_mcp_global_policy.py#L111)

Should we use similar notes and descriptions in this module?

Reply:
Sounds good to me!